package.json

```diff
@@ -12,13 +12,14 @@
   "main": "js/index.js",
   "license": "BSD-2-Clause",
   "scripts": {
-    "clean": "rimraf ./dist/*",
+    "clean": "rm -rf ./dist/*",
     "prebuild": "npm run clean",
     "build": "NODE_ENV=production webpack --optimize-minimize",
     "test": "TZ=UTC NODE_ENV=test karma start",
     "browser-tests": "karma start --browsers Chrome",
     "test-watch": "TZ=UTC NODE_ENV=test karma start --no-single-run --reporters=mocha",
     "start": "NODE_ENV=development webpack-dev-server -d --cache --colors --progress --content-base example --port 8081",
+    "security-checks": "npm audit --production --registry=https://registry.npmjs.org/",
```

Comment:
did we not once say we want to restrict the audit level to moderate?
--audit-level=moderate

Reply:
Was just for me, to have an idea, not used yet.
anyway, as of today:
```
> npm audit --production --registry=https://registry.npmjs.org/


                       === npm audit security report ===

found 0 vulnerabilities
 in 113 scanned packages
```